keyboards/jopr/jopr.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
+    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K600, K601, K602, K603, K604, K605, K606, K607, \
```

Comment:
Ideally, Lines 6-11 here would resemble the physical arrangement of the keys on the actual keyboard, though this isn't strictly required.

Reply:
done